mezzrow/oscar_apps/checkout/forms.py

```diff
@@ -16,6 +20,19 @@ def clean(self):
                 del self.errors['first_name']
             if 'last_name' in self.errors:
                 del self.errors['last_name']
+        if self.is_guest_checkout() or self.is_new_account_checkout():
+            if 'password' in self.errors:
+                del self.errors['password']
+            if 'username' in self.cleaned_data:
+                email = normalise_email(self.cleaned_data['username'])
```

Comment:
because it helps fulfill the request for a back-end list of email address collected. Oscar's guest checkout only captures the PayPal account email address, not the address put in at the beginning. This allows the customer management tab to show all emails.